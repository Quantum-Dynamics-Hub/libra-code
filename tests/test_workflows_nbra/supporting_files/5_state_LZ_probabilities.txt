0.96 0.05 0.04 0.03 0.02
0.01 0.80 0.16 0.10 0.08
0.01 0.08 0.70 0.20 0.10
0.01 0.04 0.06 0.60 0.30
0.01 0.03 0.04 0.07 0.50
