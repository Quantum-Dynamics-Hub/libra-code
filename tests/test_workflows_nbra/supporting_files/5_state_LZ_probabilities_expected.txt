0.96 0.05 0.04 0.03 0.02
0.01 0.80 0.16 0.10 0.00
0.01 0.08 0.70 0.00 0.10
0.01 0.04 0.00 0.60 0.30
0.01 0.00 0.04 0.07 0.50
